pypeit/spectrographs/keck_lris.py

```diff
@@ -346,6 +398,44 @@ def check_frame_type(self, ftype, fitstbl, exprng=None):
 
         msgs.warn('Cannot determine if frames are of type {0}.'.format(ftype))
         return np.zeros(len(fitstbl), dtype=bool)
+
+    def vet_assigned_ftypes(self, type_bits, fitstbl):
```

Comment:
We should note somewhere that this function should only be called when running `pypeit_setup`.  I.e., we don't want this to be used to overwrite any user-provided frame types.  I think that's true now, but I think we should add some warning comments to our future selves about this.